FlorenceBML/BMLor321/BMLor321.xml

```diff
@@ -34,8 +34,133 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                         <altIdentifier>
                             <idno>Marrassini ms. 21</idno>
                         </altIdentifier>
+                        <altIdentifier>
+                            <idno>Assemani CCCCV</idno>
+                        </altIdentifier>
                     </msIdentifier>
+                    <msContents>
+                        <summary/>
+                        <msItem xml:id="ms_i1">
+                            <locus from="1r" to="68v"/>
+                            <title>Gǝʿǝz-Latin Lexicon</title>
+                            <note>According to <bibl><ptr target="bm:bm:Assemani1719Bibliotheca"/><citedRange unit="page">431</citedRange>
+                                </bibl> the author is <persName ref="PRS14230GBRaimondi"/>.
+                                The Vocabulary is apparently taken from lyrics and arranged by initial letter, but many pages remain blank, i.e. the work is 
+                                largely unfinished. Many Ethiopian words are given in a status construct, causatives or brokan plural. The succession is 
+                                different from the traditional order and may be seen as a modified ʾabugidā-type: <foreign xml:lang="gez">አ፡ በ፡ ገ፡ ደ፡ ሀ፡ ወ፡ 
+                                ዘ፡ ሐ፡ ኀ፡ ጠ፡ የ፡ ከ፡ ለ፡ ነ፡ ሰ፡ ዐ፡ ጰ፡ ፈ፡ ጸ፡ ፀ፡ ቀ፡ ረ፡ ሠ፡ ተ፡ ፐ፡ መ፡</foreign>.
+                                Contrary to <bibl><ptr target="bm:bm:Assemani1719Bibliotheca"/><citedRange unit="page">431</citedRange></bibl> 
+                                the work does not contain grammatical parts.</note>
+                        </msItem>
+                    </msContents>
+                    <physDesc>
+                        <objectDesc form="Codex">
+                            <supportDesc>
+                                <support>
+                                    <material cert="medium" key="paper"/>
+                                </support>
+                                <extent>
+                                    <measure unit="leaf" quantity="134">189</measure> 
```

Comment:
could you explain the difference between 134 and 189?

Reply:
Copy-paste error - it is 189 in fact.